src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -650,6 +655,15 @@ subroutine ocn_tend_tracer(tendPool, statePool, forcingPool, diagnosticsPool, me
                modifiedGroupName = trim(groupItr % memberName) // "Tend"
                call mpas_pool_get_array(tracersTendPool, trim(modifiedGroupName), tracerGroupTend)
 
+               modifiedGroupName = trim(groupItr % memberName) // "_redi_term1"
+               call mpas_pool_get_array(redi1Pool, trim(modifiedGroupName), redi_term1)
+
+               modifiedGroupName = trim(groupItr % memberName) // "_redi_term2"
+               call mpas_pool_get_array(redi2Pool, trim(modifiedGroupName), redi_term2)
+
+               modifiedGroupName = trim(groupItr % memberName) // "_redi_term3"
+               call mpas_pool_get_array(redi3Pool, trim(modifiedGroupName), redi_term3)
+
```

Comment:
these output the individual redi terms (except vertical mixing). we can revert these when we want to merge.